.travis.yml

```diff
@@ -9,7 +9,7 @@ install:
   - npm install
   - npm run frontend-build:production
 script:
-  - ./scripts/run_tests.sh --cov=app --cov-report=term-missing
+  - PYTEST_ARGS='--cov=app --cov-report=term-missing' make test
```

Comment:
This is the script run by travis, I don't think there are any additional arguments passed to it, so this should be fine.